modules/layers/src/path-layer/path-layer.js

```diff
@@ -120,7 +120,7 @@ export default class PathLayer extends Layer {
       if (this.state.model) {
         this.state.model.delete();
       }
-      this.setState({model: this._getModel(gl)});
+      this.setState({model: gl && this._getModel(gl)});
```

Comment:
More readable/likely to avoid issues in other places if we push the gl check inside `_getModel`?
 
If `_getModel` is the one sublayers override maybe call it through `getModel` and check there?